SU-SWS/saa_victoria_subtheme
# ADAPT-476: Rebuilt master 1.x branch.

# READY FOR REVIEW

# Summary
- realized theme.css was not updated after merges into 1.x

# Review By (Date)
- 7/17

# Urgency
- need to fix soon

# Steps to Test

1. pull down
1. clear cache
2. notice that the top banner with text and image should have bright red background, not vivid red. 

# Affected Projects or Products
- n/a

# Associated Issues and/or People
- JIRA ticket 476 testing for small issues
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
